File: src/main/java/com/jiaoay/rime/core/RimeStatus.java
package com.jiaoay.rime.core;

/**
 * Rime狀態
 */
public class RimeStatus {
    int data_size;
    // v0.9
    String schema_id;
    String schema_name;
    boolean is_disabled;
    boolean is_composing;
    boolean is_ascii_mode;
    boolean is_full_shape;
    boolean is_simplified;
    boolean is_traditional;
    boolean is_ascii_punct;
}

File: src/main/java/com/jiaoay/rime/core/RimeStatus.kt
package com.jiaoay.rime.core

/**
 * Rime狀態
 */
class RimeStatus {
    var data_size = 0

    // v0.9
    var schema_id: String = ""
    var schema_name: String = ""
    var is_disabled = false
    var is_composing = false
    var is_ascii_mode = false
    var is_full_shape = false
    var is_simplified = false
    var is_traditional = false
    var is_ascii_punct = false
}